nilearn/plotting/img_plotting.py

```diff
@@ -314,6 +314,7 @@ class _MNI152Template(SpatialImage):
     affine = None
     vmax = None
     _shape = None
+    header = None
```

Comment:
Can you give an explanation here ?

Reply:
Without it, the following error is raised (see b9e15b9da31faa5fa9b607f8d541a8c273e32a20): 

AttributeError: '_MNI152Template' object has no attribute '_header'

The check I propose in load_niimg checks to see if the header is None. But MNI152Template has no header at all, this gives it a header.

Reply:
Simply add a short comment: "Having a header is required by the load_niimg function"

Reply:
Done